Week-3/assignment-2/bind.js

```diff
@@ -0,0 +1,16 @@
+const printUser = {
```

Comment:
@chinmay4o please add some comments explaining it as well

Reply:
added